knode:
  ros__parameters:
    Kp_linear: 9.4
    Kd_linear: 0.1
    Kp_angular: 7.7
    Kd_angular: 0.05